app/routes.py

```diff
@@ -1,2 +1,272 @@
-from flask import Blueprint
+from flask.wrappers import Response
+from app.models.task import Task
+from app import db
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import date
+import os
+import requests
+from dotenv import load_dotenv
+from app.models.goal import Goal
+
+# handle_tasks handles GET and POST requests for the /tasks endpoint
+
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+def valid_int(number,parameter_type):
+    try:
+        int(number)
+    except:
+        abort(make_response({"error":f"{parameter_type} must be an int"},400))
+
+@tasks_bp.route("", methods=["GET", "POST"])
+def handle_tasks():
+# Wave 1: Get Tasks: Getting Saved Tasks
+    if request.method == "GET":
+        sort = request.args.get("sort")
+        if sort == "asc":
+            tasks = Task.query.order_by(Task.title)
+        elif sort == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+        else:
+            tasks = Task.query.all()
+#Wave 1: Get Tasks: No Saved Tasks
+        tasks_response = []
+        for task in tasks:
+            has_complete = task.completed_at
+            tasks_response.append(
+                {
+                    "description": task.description,
+                    "id": task.id,
+                    "is_complete": False if has_complete == None else has_complete,
+                    "title": task.title,
+                }
+            )
```

Comment:
This works, I recommend using the to_dict helper function to DRY this section:
```suggestion
            tasks_response.append(task.to_dict())
```